src/parser/parse.ts

```diff
@@ -469,6 +475,73 @@ function parseGroupOpen(token: GroupOpenToken, context: Context, state: State):
   return node;
 }
 
+function parseNamedCallout({kind, name, tag, args}: NamedCalloutToken): NamedCalloutNode {
+  const argumentValues: Array<string | number> =
+    args ?
+      args.split(',')
+        .filter((argument: string): boolean => argument !== '') // oniguruma skips over/ignores redundant/unnecessary commas
```

Comment:
nitpick: I try to avoid adding long comments to the right of code, so prefer:

```js
// Onig skips over/ignores extra commas
.filter(arg => argument !== '')
```

Also, my style throughout this library has been to make automatic-semicolon-insertion (or lack of it) super clear by always putting continuation syntax (like `.`) on the preceding line. Hence:

```js
args.split(',').
  filter(...).
  map(...)
```

In this case maybe:

```js
args ?
  args.split(',').
    // Onig skips over/ignores extra commas
    filter(arg => arg !== '').
    map(arg => /^[+-]?\d+$/.test(arg) ? +arg : arg) :
  [];
```

(I think all the types are implicitly understood by TS here.)